server/core/config/raw/autodiscover_test.go

```diff
@@ -0,0 +1,131 @@
+package raw_test
+
+import (
+	"testing"
+
+	"github.com/runatlantis/atlantis/server/core/config/raw"
+	"github.com/runatlantis/atlantis/server/core/config/valid"
+	. "github.com/runatlantis/atlantis/testing"
+	yaml "gopkg.in/yaml.v2"
+)
+
+func TestAutoDiscover_UnmarshalYAML(t *testing.T) {
+	auto_discover_enabled := valid.AutoDiscoverEnabledMode
```

Comment:
can you change this to `autoDiscoverEnabled` variables with underscores are not used much in golang

Reply:
please look at the linting error and address the comments otherwise it looks good 
thanks @jskrill

Reply:
Made all the snake case variable names camel case and fixed the other linting errors.